ts/modules/exporting.src.ts

```diff
@@ -1993,9 +1993,10 @@ extend(Chart.prototype, /** @lends Highcharts.Chart.prototype */ {
                             },
                             innerHTML: (
                                 (item as any).text ||
-                                (chart.options.lang as any)[
-                                    (item as any).textKey as any
-                                ]
+                                (chart.dataTableDiv && item.textKey === 'viewData') ?
```

Comment:
Instead of making decision here, can we change `item.textKey` when table is being generated/hidden? For example `viewData()` can change `textKey` to `viewData` and `hideData()` can change to `hideData`.